recipe/gen_patch_json.py

```diff
@@ -1594,6 +1595,16 @@ def _gen_new_index_per_key(repodata, subdir, index_key):
         if record_name == "xarray" and record["version"] == "0.19.0":
             _replace_pin("python >=3.6", "python >=3.7", deps, record)
 
+        # Xarray <=2023.1.0 doesn't support pandas 2.0 yet.
+        # https://github.com/pydata/xarray/issues/7716
+        if record_name == "xarray" and packaging.version.Version(record["version"]) < packaging.version.Version("2023.3.0"):
```

Comment:
`2023.02.0` should be `pandas >= 1.4,<2`

https://github.com/pydata/xarray/blob/v2023.02.0/requirements.txt

I'm confused why that doesn't get listed.

Reply:
The pattern tries to catch only `pandas >=1.3`. I added another if-clause for `xarray 2023.2.0`.